tests/unit/-private/properties/alias-test.js

```diff
@@ -3,31 +3,35 @@ import {
   create,
   clickable,
   collection,
-  isVisible
+  isVisible,
+  text
 } from 'ember-cli-page-object';
-import { alias } from 'ember-cli-page-object/macros';
+import {
+  alias,
+  test as testMacro
```

Comment:
Having to rename `test` was a bit rough; I forgot to type `testMacro` several times when writing the tests below, and it took me a few minutes to figure out what I was doing wrong each time. Do we think this will cause any issues for people? Seems unlikely (because I'm not sure how you'd end up with two `test` values in a PageObject), but I wanted to raise the question.